webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
```

Comment:
@henbos  I disagree. These foreign-keys are references, and leaving broken references leads to bugs. Had we had a different API where it was possible to look up these references with another call, then I might have agreed, but we don't.